config/database.yml

```diff
@@ -14,4 +14,5 @@ test:
   database: <%= ENV.fetch('DATABASE_NAME', 'timeoverflow_test') %>
 
 production:
-  url: <%= ENV.fetch('DATABASE_URL', 'Set DATABASE_URL environment variable') %>
+  # Set DATABASE_URL environment variable
+  url: <%= ENV.fetch('DATABASE_URL', 'null') %>
```

Comment:
If we remove only that line, it is invalid YAML format. so either we skip the production section (and in fact in actual production there is `ENV['DATABASE_URL']` exactly for that reason) or we leave a section just for completeness, there was already a conversation about that, and we ended up adding that section for the [POLA](https://en.wikipedia.org/wiki/Principle_of_least_astonishment).